### PR TITLE
[FIX] improve longitudinal example

### LINKED ISSUE #1575: [BUG] Longitudinal example doesn't include a `*_sessions.tsv` example
### Describe your problem in detail.

The longitudinal example is incomplete, lacking a `*_sessions.tsv` file in the file directory structure, as well as an example of the file's contents given the session setup provided as an example.

### Describe what you expected.

An example sessions.tsv file to complete the longitudinal example.

### BIDS specification section

https://bids-specification.readthedocs.io/en/stable/06-longitudinal-and-multi-site-studies.html

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #1575

### CODE REVIEW

**Comment by Remi-Gau** [comment]
@gdevenyi does this look better?

https://bids-specification--1576.org.readthedocs.build/en/1576/longitudinal-and-multi-site-studies.html#longitudinal-and-multi-site-studies

**Review by sappelhoff — APPROVED** [review]

File: src/longitudinal-and-multi-site-studies.md [hunk 1]
```diff
@@ -62,6 +63,19 @@ A guide for using macros can be found at
     }
 ) }}
 
+`sub-control01_sessions.tsv` content:
+
+```Text
+session_id	acq_time	systolic_blood_pressure
+ses-predrug	2009-06-15T13:45:30	120
```
Comment (effigies) [inline src/longitudinal-and-multi-site-studies.md:70]: Should we use anonymized (pre-1925) times?

Comment (Remi-Gau) [reply src/longitudinal-and-multi-site-studies.md:70]: I grabbed it from the sessions.tsv section: should we anonymize in both places?

Comment (effigies) [reply src/longitudinal-and-multi-site-studies.md:70]: Ah, if we're reusing, I'm cool with maintaining consistency.

**Comment by gdevenyi** [comment]
Looks good, thanks!

**Review by effigies — APPROVED** [review]

**Pull request merged**